src/System.Net.Requests/tests/FileWebRequestTest.cs

```diff
@@ -80,7 +80,7 @@ public void GetRequestResponseAfterAbort_Throws()
         [Fact]
         public void NotImplementedMembers_Throws()
         {
-            WebRequest request = WebRequest.Create("file://anything");
+            WebRequest request = WebRequest.Create("file:///anything");
```

Comment:
Why does that have to change?  I'm concerned about this being a breaking change.  Are there no valid situations someone could have been using this today without the extra '/'?